plugins/azure-devops-backend/api-report.md

```diff
@@ -0,0 +1,70 @@
+## API Report File for "@backstage/plugin-azure-devops-backend"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Build } from 'azure-devops-node-api/interfaces/BuildInterfaces';
+import { BuildResult } from 'azure-devops-node-api/interfaces/BuildInterfaces';
+import { BuildStatus } from 'azure-devops-node-api/interfaces/BuildInterfaces';
+import { Config } from '@backstage/config';
+import express from 'express';
+import { GitRepository } from 'azure-devops-node-api/interfaces/GitInterfaces';
+import { Logger as Logger_2 } from 'winston';
+import { WebApi } from 'azure-devops-node-api';
+
+// Warning: (ae-missing-release-tag) "AzureDevOpsApi" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export class AzureDevOpsApi {
```

Comment:
Ideally this would be an interface, but tbh it prolly doesn't matter too much for this and we can change that in the future if needed

Reply:
Sounds good, I'm new to most of this, is there an example I could look over that would help me move from a class to an interface? Thanks in advance!

Reply:
Sure! The todo-backend is a pretty slim example https://github.com/backstage/backstage/blob/92c3d6dc0441ed0cc9d129476b936c9aa3558194/plugins/todo-backend/src/service/router.ts#L32